src/main/java/com/minecolonies/coremod/blocks/BlockMinecoloniesRack.java

```diff
@@ -175,19 +186,47 @@ public IBlockState getActualState(final IBlockState state, final IBlockAccess wo
     }
 
     /**
-     * Check if a certain block should be replaced with a rack.
-     * @param block the block to check.
-     * @return true if so.
+     * Convert the BlockState into the correct metadata value.
+     *
+     * @deprecated (Remove this as soon as minecraft offers anything better).
      */
-    public static boolean shouldBlockBeReplacedWithRack(final Block block)
+    @NotNull
+    @Override
+    @Deprecated
+    public IBlockState withRotation(@NotNull final IBlockState state, final Rotation rot)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1133)